tests/test_import_data.py

```diff
@@ -18,9 +18,18 @@ def test_attachment_ftp_to_nowhere():
     assert "urlopen error" in str(e.value)
 
 
+def convert_to_path_with_tilde(string_path):
+    path = pp.Path(string_path)
+    absolute_path = path.resolve()
+    home = absolute_path.home()
+    string_path_with_tilde = str(absolute_path).replace(str(home), '~')
+    return string_path_with_tilde
```

Comment:
(I suppose you just got this from `SubmitCGAP` so maybe I should look at applying these same comments there.)

A doc string (e.g., that notes this is the inverse of `os.path.expanduser`) would be useful.

I am a little worried about why we would ever want to use this, though. Tilde is handled in bash as a user-interface matter because the semantics of a 'relative to the current user' filename is weird. It means something semantically different in different situations, but unlike relative pathname, it's outside of the user's ability to control. I'm uncertain as to why relative names are not better if that's what you want. If the user wants to be in their home directory, they can.

If a user uploaded it from a specific place, and if it's important to say where in the hierarchy (something I'm not sure it is) saying it came from their actual directory name seems relevant. If they want to download it, you're only guessing that ~/test/foo.whatever is a good place. They might prefer ~/cgap-stuff/test/foo.whatever but have no way to change it. Whereas if it was stored as a relative name, they can put their working directory where they want and still download it to the right place.